.mage/tools.mk

```diff
@@ -2,16 +2,30 @@ mage_folder := $(abspath $(dir $(lastword $(MAKEFILE_LIST))))
 mage_generated_path := $(mage_folder)/gen
 mage_targets_file := $(mage_generated_path)/targets.mk
 mage := $(mage_generated_path)/local-mage
+mgmake := $(mage_folder)/mgmake.go
+
+define mgmake_content
+// +build mage
```

Comment:
do we want a Go 1.17 build directive here too?

Reply:
Yep, fixed!